### DISCUSSION TITLE
研究生-Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by GZ-Metal-Cell** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#299] [Announcements]
# 研究生 | Zi-Zi's Journey

Zi-Zi's Journey

https://gz-metal-cell.github.io/galleries/%E7%A0%94%E7%A9%B6%E7%94%9F/

**Comment by promefire** [comment]
小迷糊是何许人也

**Comment by GZ-Metal-Cell** [comment]
> @promefire
> 小迷糊是何许人也

本科同学，武汉的🥱技术吊锤 promefire

**Comment by promefire** [comment]
催更广州相册

**Reply by GZ-Metal-Cell** [reply]
这个相册已经年久失修了😭
